verify.ts

```diff
@@ -141,7 +141,7 @@ const processContracts = async (): Promise<any> => {
           case 'DAORegistry':
             constructorArguments =
               abi.rawEncode(['address'],
-                [options.network === 'mainnet' ? '0xd3BA32dd207Db75f535001FAC749c925423D8A6f' :  // DAOstack multisig :
+                [options.network === 'mainnet' ? '0x85e7fa550b534656d04d143b9a23a11e05077da3' :  // DAOstack controlled account
```

Comment:
how this is related to the verifier ?

Reply:
When you verify a contract you have to supply the arguments (if there are any) by which it was constructed at the given address.

Reply:
right. as comment above  you should read the deployed byte code from the blockchain , derived the constructor params and then verify.

Reply:
This is done.  It adds a new "-p" option to the command line to supply a provider url for web3.

The technique I used is simple but presumes to know the _types_ (not the values) of contructor parameters in order to know how to parse the bytecodes to obtain the param values.